src/main/resources/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategy/config.jelly

```diff
@@ -45,11 +46,12 @@ THE SOFTWARE.
     <f:checkbox />
   </f:entry>
   </j:if> <!-- authorizeProjectContext -->
-  </table></f:block>
+  </table>
   <j:if test="${authorizeProjectContext != 'global'}">
   <st:once>
     <st:adjunct includes="org.jenkinsci.plugins.authorizeproject.strategy.SpecificUsersAuthorizationStrategy.checkPasswordRequested" />
     <st:adjunct includes="org.jenkinsci.plugins.authorizeproject.strategy.SpecificUsersAuthorizationStrategy.passwordApiTokenSwitch" />
   </st:once>
   </j:if> <!-- authorizeProjectContext -->
+</f:block>

```

Comment:
Issue described in jenkinsci/workflow-cps-plugin#7 and jenkinsci/jenkins-test-harness@acf3d3772cbd23a10283dec87988b90f87c05185.